source/as-stream.js

```diff
@@ -14,6 +14,8 @@ module.exports = options => {
 
 	options.gotRetry.retries = () => 0;
 
+	options.gotRetry.retries = () => 0;
```

Comment:
Isn't this the same as line 15?

Reply:
Yes, I forgot to remove it when I was resolving a merge conflict.